### DISCUSSION TITLE
Handling many files in s3 bucket

### DISCUSSION THREAD

**Discussion by simonebenati** [longhorn/longhorn#8505] [General]
Hello,

We're encountering this issue with longhorn1.4.3 where we are perfectly capable of listing in our s3 bucket via client (i.e. cyberduck) but once we try via longhorn it always timeouts. both from the manager pod and inside the ui in the backup tab.
Is there a way to increase the timeout, i can't find anything in the docs? Or how can we handle this situation simply decrease the retention? Also how could we recover from this situation?
The bucket is around 800Gb big.

Thank you

**Comment by derekbit** [comment]
cc @mantissahz

**Comment by simonebenati** [comment]
We find this to be failing even after adding an entirely new and empty s3 bucket. Other apps that offload to s3 work correctly for example kasten or rancher etcd backup..
